src/coordinator/roles.js

```diff
@@ -161,6 +257,88 @@ class Roles {
       .get(`${this._baseUrl}/organizations/${organizationId}/roles`)
       .then((roles) => toCamelCase(roles));
   }
+
+  /**
+   * Remove an application from a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: DELETE
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} applicationId The UUID formatted ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.roles.delete('4f0e51c6-728b-4892-9863-6d002e61204d');
+   */
+  removeApplication(organizationId, roleId, applicationId) {
+    if (!organizationId) {
+      return Promise.reject(
+        new Error('An organizationId is required for removing an application from a role.')
+      );
+    }
+
+    if (!roleId) {
+      return Promise.reject(
+        new Error('A roleId is required for removing an application from a role.')
+      );
+    }
+
+    if (!applicationId) {
+      return Promise.reject(
+        new Error('An applicationId is required for removing an application from a role.')
+      );
+    }
+
+    return this._request.delete(
+      `${this._baseUrl}/applications/${applicationId}/roles/${roleId}`
+    );
+  }
+
+  /**
+   * Remove an stack from a role
+   *
+   * API Endpoint: '/stacks/:stacks_id/roles/:roleId'
+   * Method: DELETE
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} stackId The UUID formatted ID of the stack
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.roles.delete('4f0e51c6-728b-4892-9863-6d002e61204d');
+   */
+  removeStack(organizationId, roleId, stackId) {
```

Comment:
Needs updated to be roles.removeStack(....) with the 3 args

Reply:
done